examples/xgboost_cv.py

```diff
@@ -0,0 +1,109 @@
+"""
+Optuna example that performs Cross Validation for cancer dataset
+using XGBoost.
+In this example, we perform cross-validation to accuracy of cancer detection
+using XGBoost. We optimize both the choice of booster model and their hyper
+parameters.
+We have following two ways to execute this example:
+(1) Execute this code directly.
+    $ python xgboost_cv.py
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize xgboost_cv.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+"""
+
+
+import optuna
+from optuna.samplers import TPESampler
+import os
+import sklearn.datasets
+import sklearn.metrics
+import xgboost as xgb
+
+
+# Set path to save CV results
+file_path = "./tmp_results_xgbCV_optuna/"
+
+# Create folder if it doesnt exist
+if not os.path.exists(file_path):
+    try:
+        os.makedirs(file_path)
+    except Exception:
+        print("please create folder to store xgboost CV results")
+
+# Set Constants
+seed = 108
+n_folds = 3
+
+# Make the Optuna Sampler behave in a deterministically.
+sampler = TPESampler(seed=seed)
```

Comment:
The most of Optuna examples do not fix seed, so please remove this line.

Reply:
I've removed this line. 
Personally, I think it's a feature that should be highlighted in the examples; it only exists in the FAQ's and having it in an example just reinforces the reproducability aspect.

Reply:
I'm sorry for the delayed response.

>  I think it's a feature that should be highlighted in the examples;

That makes sense. The examples in this repository tend to prefer simplicity to the completeness (including reproducibility). For now, I'd like to follow the convention, but I appreciate your suggestion.